recipe-server/normandy/base/utils.py

```diff
@@ -70,7 +70,7 @@ def filter_m2m(qs, field, values):
     return qs
 
 
-def sri_hash(data):
+def sri_hash(data, url_safe=False):
```

Comment:
I'd argue that we are in the context of subresource integrity here. This usage isn't covered by the SRI spec, but we are using the hashes in the same way for the same purpose: verifying that we can trust remote servers to serve the correct data. Although it is true that this is simply a hash, it's purpose is something specific that gives more meaning than just a hash.